app/html/index.html

```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
-<html lang="en" ng-app="demoApp">
+<html lang="en" ng-app="appShooter">
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Shooter-Log-FE</title>
   <script src="bundle.js"></script>
   <link rel="stylesheet" href="bundle.css">
+  <link rel='icon' href='grey-logo.png' type='image/png'>
 </head>
 <body>
-  <h3>shooter-log-pe-scaffolding</h3>
+  <app-menu></app-menu>
```

Comment:
Its great to see you have a feature out side of the ng-view this is very smart, and makes it so you have less redudent code :)
great work